perun-core/src/main/java/cz/metacentrum/perun/core/impl/PerunRolesLoader.java

```diff
@@ -247,7 +248,18 @@ private Set<PerunPolicy> loadPoliciesFromJsonNode(JsonNode rootNode) {
 			List<String> includePolicies = new ArrayList<>(objectMapper.convertValue(policyNode.get("include_policies"), new TypeReference<List<String>>() {
 			}));
 
-			policies.add(new PerunPolicy(policyName, perunRoles, includePolicies));
+			List<Map<String,String>> mfaRules = new ArrayList<>();
+			JsonNode perunMFARulesNode = policyNode.get("mfa_rules");
+			// remove if after mfa rules added
```

Comment:
just a thought - maybe we could skip the whole `mfa_rules` section in case it is not critical operation? it would save us some time setting it empty for every non-critical method. and then the `if` here would be valid check